docs/cookbooks/studio/dynamic-quickreplies.md

```diff
@@ -58,3 +57,30 @@ return new Promise(resolve => {
 });    
 ```
 
+## Create dynamic WhatsApp list 
+
+To create dynamic WhatsApp list, you need follow the same procedure as dynamic quick replies.
+
+You need use the below sample snippet to generate dynamic WhatsApp list:
```

Comment:
Hey a small suggestion, you can say 'Use the below snippet to generate a dynamic Whatsapp list'.